src/cmd/sign.js

```diff
@@ -107,6 +110,19 @@ export default function sign(
         downloadDir: artifactsDir,
       });
 
+      if (updateLink && updateLink.length > 0 && signingResult.success) {
+        let extensionID;
+        if (signingResult.id) {
+          extensionID = signingResult.id;
+        } else {
+          extensionID = id;
+        }
+        await generateNewUpdateManifest(
+          extensionID, manifestData.version, signingResult.downloadedFiles[0],
```

Comment:
I understand that's the case, but how would that fit inside an updateManifest.json file? Here's what I don't get:

id: 123456789
version: 1.2
android: someGeneratedXPI-ANDROID-v1.2.xpi
desktop: someGeneratedXPI-DESKTOP-v1.2.xpi

The older updateManifest.json file looks like this
```
{
    "addons": {
        "123456789": {
            "updates": []
        }
    }
}
```

So let's update it with the given data

```
{
    "addons": {
        "123456789": {
            "updates": [
                {
                    "version": "1.2",
                    "update_link": "https://example.com/someGeneratedXPI-ANDROID-v1.2.xpi"
                },
                {
                    // also version 1.2?
                    // this shouldn't be possible, right?
                    // the android XPI has the same extensionID, no?
                    "version": "1.2",
                    "update_link": "https://example.com/someGeneratedXPI-DESKTOP-v1.2.xpi"
                }
            ]
        }
    }
}
```